src/fuzz.zig

```diff
@@ -1,11 +1,14 @@
 const std = @import("std");
 const lib = @import("./lib.zig");
+const config = @import("./cmd/config.zig");
```

Comment:
Or should we configure the metrics port another way?

Reply:
i think this is ok for now